btree/btree.c

```diff
@@ -4633,6 +4700,7 @@ static int64_t btree_del_resolve_underflow(struct m0_btree_op *bop)
 		}
 		node_seq_cnt_update(lev->l_node);
 		node_fix(node_slot.s_node, bop->bo_tx);
+		btree_callback_add(oi->i_callback, lev->l_node);
```

Comment:
you might need to add btree_callback_add(oi->i_callback, lev->l_node); and btree_callback_add(root_child); as well at line no.: 4746 or you might need to add btree_callback_add() in node_move for src and tgt node

Reply:
Since I do not have oimpl in node_move, added callbacks after node_move for root_child and lev->l_node.